frontend/src/pages/MindMapPage.vue

```diff
@@ -0,0 +1,178 @@
+              { id: 'child2aiii', topic: 'RFC 3339 (date time)' },
+              { id: 'child2aiv', topic: 'CMIXF-12 (units)' },
+            ],
+          },
+          { id: 'child2b', topic: 'HED' },
+          {
+            id: 'child2c',
+            topic: 'JSON',
+            expanded: false,
+            children: [{ id: 'child2ci', topic: 'YAML' }],
+          },
+          { id: 'child2d', topic: 'Markdown' },
+          { id: 'child2e', topic: 'NIfTI' },
+          {
+            id: 'child2f',
+            topic: 'NWB',
+            expanded: false,
+            children: [{ id: 'child2fi', topic: 'HDF5' }],
+          },
+          { id: 'child2g', topic: 'OME Zarr' },
+          { id: 'child2h', topic: 'TSV' },
+        ],
+      },
+      {
+        id: 'child6',
+        topic: 'BEPs',
+        expanded: false,
+        children: [
+          { id: 'child6a', topic: 'BEP001' },
+          { id: 'child6b', topic: 'BEP026' },
+        ],
+      },
+      {
+        id: 'child3',
+        topic: 'Metadata',
+        expanded: false,
+        children: [
+          { id: 'child3a', topic: 'DICOM' },
+          { id: 'child3b', topic: 'HED' },
+          { id: 'child3c', topic: 'Cognitive Atlas' },
+          { id: 'child3d', topic: 'CogPo' },
+        ],
+      },
+      {
+        id: 'child5',
+        topic: 'File system',
+        expanded: false,
+        children: [
+          { id: 'child5a', topic: 'Filename (>8.3)' },
+          { id: 'child5b', topic: 'Folder (circa 1958)' },
+        ],
+      },
+      {
+        id: 'child4',
+        topic: 'Automation/CI',
+        expanded: false,
+        children: [
+          {
+            id: 'child4a',
+            topic: 'Circle CI',
+            expanded: false,
+            children: [
+              { id: 'child4ai', topic: 'build' },
+              { id: 'child4aii', topic: 'linkcheck' },
+            ],
+          },
+          {
+            id: 'child4b',
+            topic: 'GitHub',
+            expanded: false,
+            children: [
+              { id: 'child4bi', topic: 'codespell' },
+              { id: 'child4bii', topic: 'no bad latin' },
+              { id: 'child4bi', topic: '...more to catch human errors...' },
+            ],
+          },
+        ],
+      },
+      {
+        id: 'child7',
+        topic: 'Network',
+        expanded: false,
+        children: [
+          {
+            id: 'child7a',
+            topic: 'HTTP',
+            expanded: false,
+            children: [
+              { id: 'child7ai', topic: 'TCP' },
+              { id: 'child7aii', topic: 'IP' },
+            ],
+          },
+          { id: 'child7b', topic: 'RFC 3986 (URI)' },
+        ],
+      },
+    ],
+  },
+  linkData: {},
+  expand: {},
+};
+
+onMounted(() => {
+  me.value = new MindElixir({
+    el: '#map',
+    direction: MindElixir.SIDE,
+  });
+  me.value.init(customData);
+});
+</script>
+<style>
+#map {
+  width: 100%;
+  height: 1000px;
```

Comment:
```suggestion
  height: calc(100vh - 195px);
```
This will calculate the height to be 100% of the view-height minus 195px.  
Goal here is to ensure the widget in the bottom right of the screen is shown and not hidden behind the footer.

Note:  195px is roughly the height of the header + footer